lib/puma/control_cli.rb

```diff
@@ -176,7 +176,8 @@ def send_request
         when 'tcp'
           TCPSocket.new uri.host, uri.port
         when 'unix'
-          UNIXSocket.new "#{uri.host}#{uri.path}"
+          UNIXSocket.new(@control_url.start_with?('unix://@') ?
```

Comment:
Everywhere that we check for a socket starting with `@`, I think we should have a comment explaining this a check for an abstract socket.

Reply:
Done